frontend/src/pages/projects/screens/spawner/SpawnerFooter.tsx

```diff
@@ -49,7 +50,13 @@ const SpawnerFooter: React.FC<SpawnerFooterProps> = ({
     !checkRequiredFieldsForNotebookStart(startNotebookData, storageData, envVariables);
   const { username } = useUser();
 
-  const redirect = () => {
+  const afterStart = (type: 'created' | 'updated') => {
+    const { gpus, notebookSize, image } = startNotebookData;
+    fireTrackingEvent(`Workbench ${type}`, {
```

Comment:
Anyway for us to test this locally? I'm not getting events fired

Reply:
hmm, I think we can make this event print to the console if we run DEV mode. WDYT?

Reply:
Hmmm, yeah I suppose that's not a bad idea.

Something like: `Telmetry event triggered: ${event info}`